### DISCUSSION TITLE
Renommer le projet "Blazing Scaffolding" ou quelque chose d'autre similaire ?

### DISCUSSION THREAD

**Discussion by Loumardes** [Gunivers/Scaffolding-Rush#47] [General]
Le projet s'éloigne de plus en plus du minijeu "Rush", donc je pense qu'il nous faudrait un nom qui évoque plus précisément le gameplay à ceux qui n'ont jamais entendu parler de "Scaffolding Rush"

J'ai eu l'idée de "Blazing Scaffolding" il y a un moment déjà et j'ai testé si des anglophones imaginait un concept proche seulement avec le nom, et ça avait l'air de marcher

![image](https://user-images.githubusercontent.com/40787527/201530100-8646ca7d-51d0-4f50-b73e-aa600e9f3bff.png)

Je pense que c'est important que les attentes suscitées par le nom du minijeu corresponde à ce qu'il est, vu que le nom est un des éléments qui suscite en premier la curiosité chez les potentiels nouveaux joueurs

Du coup, qu'est ce que vous en pensez ?

**Comment by VForiel** [comment]
Le terme "Rush" à l'avantage de susciter un grand intérêt (cd. le nombre de DL sur le projet Rush Reloaded sans avoir fait plus de comm que ça)

Ceci étant, l'argument des attentes est également vrai ... dilemme donc 😅

**Reply by TuTurDuDur** [reply]
On peut mettre les trois
